nb2kg/handlers.py

```diff
@@ -86,8 +89,21 @@ def get(self, kernel_id, *args, **kwargs):
         self.kernel_id = cast_unicode(kernel_id, 'ascii')
         yield gen.maybe_future(super(WebSocketChannelsHandler, self).get(kernel_id=kernel_id, *args, **kwargs))
 
+    def send_ping(self):
+        if self.ws_connection is None and self.ping_callback is not None:
+            self.ping_callback.stop()
+            return
+
+        self.ping(b'')
+
+    def on_pong(self, data):
+        self.log.debug('Client has sent pong')
```

Comment:
I see the Notebook code implements a timeout mechanism where `on_pong` captures the last pong time.  Is that not necessary here (between Notebook and Gateway)?  If not, can we remove this method of comment out the debug statement and use `pass`?

Reply:
I don't think we need it because KG already have logic for culling idle kernel. It will cover culling websocket of no response.